lib/node_modules/@stdlib/math/base/special/asinf/package.json

```
@@ -0,0 +1,69 @@
+{
+  "name": "@stdlib/math/base/special/asinf",
+  "version": "0.0.0",
+  "description": "Compute the arcsine of a single-precision floating-point number.",
+  "license": "Apache-2.0",
+  "author": {
+    "name": "The Stdlib Authors",
+    "url": "https://github.com/stdlib-js/stdlib/graphs/contributors"
+  },
+  "contributors": [
+    {
+      "name": "The Stdlib Authors",
+      "url": "https://github.com/stdlib-js/stdlib/graphs/contributors"
+    }
+  ],
+  "main": "./lib",
+  "directories": {
```

```suggestion
  "gypfile": true,
  "directories": {
```

Missing field to inform package managers that there is a GYP file.

---

This is only applicable for packages with native add-ons.